From a83b750a9fad2528f078e92bce84637134b404d1 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Mon, 13 Jun 2016 09:13:20 +0800
Subject: [PATCH] arm64: dts: rockchip: add arm,pl330-broken-no-flushp quirk
 for rk3368

Pl330 integrated in rk3368 platform doesn't support
DMAFLUSHP function. So we add arm,pl330-broken-no-flushp quirk
for it.

Change-Id: Ia7f4bb6ffa1fba01dac5ac2257499dbbc9887da6
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368.dtsi b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
index e73d619823ee..f3ad29070dc5 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
@@ -284,6 +284,7 @@
 			#dma-cells = <1>;
 			clocks = <&cru ACLK_DMAC_PERI>;
 			clock-names = "apb_pclk";
+			arm,pl330-broken-no-flushp;
 		};
 
 		dmac_bus: dma-controller@ff600000 {
@@ -294,6 +295,7 @@
 			#dma-cells = <1>;
 			clocks = <&cru ACLK_DMAC_BUS>;
 			clock-names = "apb_pclk";
+			arm,pl330-broken-no-flushp;
 		};
 	};
 
-- 
2.35.3

